README.md

```diff
@@ -0,0 +1,3 @@
+# AMITY
+
```

Comment:
Upadate the README to include info for the app, setting up, what it does, and how to set up